### DISCUSSION TITLE
[Update-DbaInstance] Error while looking for SQL Server installations on <---ComputerName--->

### DISCUSSION THREAD

**Discussion by DavidRaphaelP** [dataplat/dbatools#8523] [Q&A]
Hello !
I want to use the Update-DbaInstance function.
When I am using, it I've got these messages:
![image](https://user-images.githubusercontent.com/65810259/188497415-cc32884e-1006-4e1c-8538-10f6cb624668.png)
I use the Install-DbaInstance function to get the Instance installed.
I setup the "Path.SQLServerUpdates" before execute the Update-DbaInstance command.
What I have to do to fix this problem ?

**Comment by potatoqualitee** [comment]
try localhost? not sure 169 ips are addressable

**Reply by DavidRaphaelP** [reply]
Thanks for your awnser, I've got the same messages (with localhost instead of $env:computername)

**Reply by potatoqualitee** [reply]
hmmm i wonder if this is a localization problem. can you use Find-DbaInstance and let me know if any instances are found? Also can you connect to that instance otherwise?

**Reply by DavidRaphaelP** [reply]
![image](https://user-images.githubusercontent.com/65810259/188576213-75855776-f263-41bb-abb5-159af1163a97.png)
I have the possibility to find it and connect to it.

**Reply by potatoqualitee** [reply]
That is helpful, thank you. You shouldn't need to but can you try giving it a hint with -InstanceName? If that works, I'll ping a dev to see how we can better get it working.

**Reply by DavidRaphaelP** [reply]
![image](https://user-images.githubusercontent.com/65810259/188590727-6c3b839e-99fb-405b-86b2-dceb17c4363d.png)
Yes same as the first time

**Reply by potatoqualitee** [reply]
Thank you! @nvarscar or @andreasjordan are either of you available to take a look at this? 🙇🏼

**Reply by andreasjordan** [reply]
Will comment in the issue #8524 .
